JSDemos/Demos/TreeList/UsingFilterRow/description.md

```diff
@@ -1 +1,32 @@
-This demo shows how you can filter data using the filter row. Its cells allow filtering by&nbsp;individual columns. Select or&nbsp;type a&nbsp;value into a&nbsp;cell of&nbsp;the filter row. If&nbsp;this value is&nbsp;found in&nbsp;any of&nbsp;the rows, the TreeList displays the full hierarchy down to&nbsp;them, but not further. In&nbsp;addition, you can choose the current filter operation from the drop-down menu that opens when you hover over the magnifying glass icon.
\ No newline at end of file
+The filter row allows users to enter a value for each column and filter grid data by those values. To display the filter row, set the **filterRow**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterRow/#visible) property to **true**. If a filter row cell should be disabled for a specific column, set the column's [allowFiltering](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFiltering) property to **false**.
+
+Depending on a column's [dataType](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#dataType), its filter row cell contains different editors:
+
+<table class="dx-table">
+    <tr>
+        <th>dataType</th>
+        <th>Editor</th>
+    </tr>
+    <tr>
+        <td>*"string"*, *"number"*, *"object"*</td>
+        <td>Text box</td>
+    </tr>
+    <tr>
+        <td>*"boolean"*</td>
+        <td>Drop-down list</td>
+    </tr> 
+    <tr>
+        <td>*"date"*</td>
+        <td>Date picker</td>
+    </tr> 
+    <tr>
+        <td>*"datetime"*</td>
+        <td>Date and time picker</td>
+    </tr> 
+</table>
+
+In this demo, the Hire Date column's filter row cell contains a date picker. Other filter row cells contain text boxes.
+
+Each data type supports its own set of filter operations. Users can click a magnifying glass icon in each filter row cell to specify the [selectedFilterOperation](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#selectedFilterOperation). If you need to restrict the available operations, specify the **columns[]**.[filterOperations](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#filterOperations) array.
```

Comment:
A data type itself supports filter operations or the tree list component supports specific filter operations for specific data types?